spec/facades/word_facade_spec.rb

```diff
@@ -2,20 +2,22 @@
 
 RSpec.describe 'Word Facade' do
   describe 'Happy Path' do
-    context 'responses with word facade', :vcr do
+    context 'responds with word facade', :vcr do
       it 'responses with a random word' do
         response = WordFacade.find_word
 
         expect(response).to be_a String
-        expect(response.split.count).to eq 1
+        expect(response.split.count).to eq (1)
       end
       
-      it 'responses with multiple random words' do
+      it 'responds with answer and 4 choices' do
```

Comment:
since this method was amended, it would be nice to see/test how the code behaves if _NO_  param is passed in.

I think this is a necessary test. 
If you do not have time to work on it please add a card to the project board.

Reply:
Will update testing to reflect different params

Reply:
Updated, see [803e03a](https://github.com/Correct-or-Crass/Trivia_API/pull/41/commits/803e03a678aa876e2a9e5803c9471d74b0e80e26)

Reply:
I like the test on L20, but it took me a second glance to find out what you were doing. perhaps the it description can be amended to comminicate something like: "responds with an 
array which contains the answer in the first index and an array with # words in the second index" or "responds with [ [answer], [words (including answer)] ]" or something like that.